packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -146,8 +146,24 @@ export class ApexTestOutlineProvider
     }
   }
 
-  public getTestClassName(uri: vscode.Uri): string | undefined {
-    return this.testIndex.get(uri.toString());
+  public getTestClassName(filePath: string): string | undefined {
+    // filePath is in the format of "file:///Users/{user-name}/{path-to-apex-file.cls}" on Mac OSX
+    // filePath is in the format of "file:///c%3A/Users/{user-name}/{path-to-apex-file.cls}" on Windows
```

Comment:
Randi ran this branch on her Windows machine this morning.  The code ran w/o any additional changes needed, and I added a comment here of what the format of `filePath` is here on Windows.